state/protocol/badger/state_test.go

```diff
@@ -149,24 +149,6 @@ func TestBootstrapNonRoot(t *testing.T) {
 	rootBlock, err := rootSnapshot.Head()
 	require.NoError(t, err)
 
-	// should be able to bootstrap from snapshot after building one block
```

Comment:
Just noting from our Slack message: this is safe to remove because it is no longer a valid test fixture, because child does not seal root.